src/lib/views/AlbumsView.svelte

```diff
@@ -211,14 +221,16 @@
                 class="grid"
                 class:show={$albums && $query.query?.length === 0}
                 class:visible={isVisible}
-                style="grid-template-columns: repeat(auto-fit, minmax({minWidth}px, 0.1fr));width: 100%;"
+                style="grid-template-columns: repeat(auto-fit, minmax({minWidth}px, 0.1fr));width: 100%;grid-auto-flow: dense;"
```

Comment:
Really cool, I've never used dense before, didn't know you could break the grid order like that